src/app/features/first-login-wizard/first-login-wizard.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="govuk-grid-row govuk-!-margin-1 govuk-!-margin-top-4">
+  <div class="asc-summary-box">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full govuk-util__align-right">
+        <a class="govuk-button govuk-button--link" href="/dashboard">Close</a>
```

Comment:
Shouldn't this be a routerlink?
```suggestion
        <a class="govuk-button govuk-button--link" routerLink="/dashboard">Close</a>
```

Reply:
Should we always use `routerLink` over `href`? I thought every `<a>` tag needed an `href`

Reply:
`routerLink` adds a `href` but wont reload the web app :)

Reply:
https://stackoverflow.com/questions/58081360/anchor-href-vs-angular-routerlink for reference